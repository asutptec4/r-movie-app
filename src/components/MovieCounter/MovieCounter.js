import PropTypes from 'prop-types';
import React from 'react';

const MovieCounter = (props) => {
  return <div className="c-p-v-15">{props.movieCount} movies found</div>;
};

MovieCounter.defaultProps = {
  movieCount: 0,
};

MovieCounter.propTypes = {
  movieCount: PropTypes.number,
};

export default MovieCounter;
