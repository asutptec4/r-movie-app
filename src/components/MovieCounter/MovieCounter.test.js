import React from 'react';
import renderer from 'react-test-renderer';

import MovieCounter from './MovieCounter';

describe('<MovieCounter />', () => {
  const props = { movieCount: 10 };
  const component = renderer.create(<MovieCounter {...props} />);

  test('should render props', () => {
    expect(component).toMatchSnapshot();
  });
});
