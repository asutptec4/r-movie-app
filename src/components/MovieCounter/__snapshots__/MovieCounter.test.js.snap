// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieCounter /> should render props 1`] = `
<div
  className="app-padding-vertical-15"
>
  10
   movies found
</div>
`;
